### PR TITLE
spearbit L-04

### PR DESCRIPTION
**Pull request by dianakocsis**
## Related Issue
Spearbit L-04
https://github.com/spearbit-audits/review-uniswap-v4/issues/18

## Description of changes
In this previous commit (https://github.com/Uniswap/v4-core/pull/837), I changed the name `feeForProtocol` to `amountToProtocol` to make it more clear that it is the actual amount, and to avoid confusion with `protocolFee` which is in pips. In this commit, I add a comment for more clarity before returning

### CODE REVIEW

**Review by snreynolds — COMMENTED** [review]
there wasn't another name change? im forgetting what we discussed

**Comment by dianakocsis** [comment]
we mentioned also changing `protocolFee` to `protocolFeePips`, but I kind of like how `lpFee`, `swapFee`, and `protocolFee` are all in pips. If we change it `protocolFeePips`, I think we should also do `lpFeePips` and `swapFeePips`

File: src/libraries/Pool.sol [hunk 1]
```diff
@@ -313,6 +314,8 @@ library Pool {
             InvalidFeeForExactOut.selector.revertWith();
         }
 
+        // 0 is the amountToProtocol (the actual fee amount going to the protocol)
```
Comment (hensha256) [inline src/libraries/Pool.sol:317]:
```suggestion
        // 0 is the fee amount that should be paid to the protocol
```

**Review by hensha256 — COMMENTED** [review]
yeah i agree i'm happy not to make that change, i feel like the comments make it clear enough

**Pull request merged**